crates/bevy_ecs/src/storage/sparse_set.rs

```diff
@@ -516,7 +528,8 @@ impl SparseSets {
         self.sets.get_mut(component_id)
     }
 
-    pub fn clear(&mut self) {
+    /// Clear entities stored in each [`SparseSet`]
+    pub fn clear_entities(&mut self) {
```

Comment:
I don't think this is publicly usable. You can't get a `&mut Storages` of any kind, cannot publicly construct `SparseSets`, and cannot get a `&mut SparseSets` from `Storages` either. We might want to make this `pub(crate)`. The same is true for the rest of the `&mut self` functions on `SparseSets`.